### PR TITLE
[Feature] Create Admin Management Table Frontend

### PR DESCRIPTION
**Pull request by aaggarwal10**
## Notion ticket link

[Build Admin Management Table Frontend](https://www.notion.so/uwblueprintexecs/ac7b1f636a214a0cbf45f6b76f899b54?v=5414ee5fc5974f149921e192d6eb29b8&p=0f020b55861d4ae2821ac075291f9c36)

## Implementation description
* Admin Management Table Frontend
* Still need to figure out dimming text on select.

Table
Three Dots Button
Role Dropdown

## Checklist
- [X] My PR name is descriptive, is in imperative tense and starts with one of the following: `[Feature]`,`[Improvement]` or `[Fix]`,
- [X] I have run the appropriate linter(s)
- [x] I have requested a review from the RCD team on GitHub, or specific people who are associated with this ticket

### CODE REVIEW

File: pages/admin/admin-management.tsx [hunk 1]
```diff
@@ -1,15 +1,140 @@
...
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  IconButton,
+  Button,
+  Select,
+} from '@chakra-ui/react'; // Chakra UI
 // Internal home page
+import { AddIcon } from '@chakra-ui/icons';
```
Comment (OustanDing) [inline pages/admin/admin-management.tsx:23]:
```suggestion
import { AddIcon } from '@chakra-ui/icons'; // Chakra UI icons
```

File: pages/admin/admin-management.tsx [hunk 2]
```diff
@@ -1,15 +1,140 @@
...
+      </MenuList>
+    </Menu>
+  );
+}
+
+type RoleProps = {
+  readonly value: string;
+};
+
+function renderRoleMenu({ value }: RoleProps) {
```
Comment (OustanDing) [inline pages/admin/admin-management.tsx:50]:
```suggestion
function renderRoleMenu({ value }: { value: string; }) {
```
And we can remove `RoleProps` above

Comment (OustanDing) [reply pages/admin/admin-management.tsx:50]: Also please add a comment to this function, for example:
```typescript
// Render the roles menu
function renderRoleMenu({ value }: { value: string; }) {
```

**Review by OustanDing — CHANGES_REQUESTED** [review]
Just some small documentation-related suggestions. Good work! Please re-request review when they're addressed :)

**Comment by OustanDing** [comment]
Note regarding table column widths: there are some issues with configuring/customizing the table widths, which was also the case for our other tables. We will keep it as it is in the screenshots above for now and address all of the tables in a future ticket.

File: pages/admin/admin-management.tsx (hunk 1 above)
Comment (aaggarwal10) [reply pages/admin/admin-management.tsx]: bff1ca1f3ef4ba015e1df9e2edba2afb0946be92

File: pages/admin/admin-management.tsx (hunk 2 above)
Comment (aaggarwal10) [reply pages/admin/admin-management.tsx]: bff1ca1f3ef4ba015e1df9e2edba2afb0946be92

File: pages/admin/admin-management.tsx [hunk 3]
```diff
@@ -1,15 +1,137 @@
...
+  GridItem,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  IconButton,
+  Button,
+  Select,
+} from '@chakra-ui/react'; // Chakra UI
 // Internal home page
```
Comment (OustanDing) [inline pages/admin/admin-management.tsx:22]:
```suggestion
```

File: pages/admin/admin-management.tsx [hunk 4]
```diff
@@ -1,15 +1,137 @@
...
+// Render the roles menu
+function renderRoleMenu({ value }: { value: string }) {
+  return (
+    <Select defaultValue={value} width={190}>
+      <option value="frontDesk">Front Desk</option>
+      <option value="accountant">Accountant</option>
+      <option value="admin">Admin</option>
+    </Select>
+  );
+}
```
Comment (OustanDing) [inline pages/admin/admin-management.tsx:55]: Can you move these functions to inside the component below, and rename them to `_renderActionsMenu` and `_renderRoleMenu`? Also please add a comment to the `_renderActionsMenu` function like that for the `_renderRoleMenu` function you had above.

File: pages/admin/admin-management.tsx [hunk 5]
```diff
@@ -1,15 +1,137 @@
...
+
+const COLUMNS = [
```
Comment (OustanDing) [inline pages/admin/admin-management.tsx:57]:
```suggestion
// Table columns
const COLUMNS = [
```

**Review by OustanDing — CHANGES_REQUESTED** [review]

File: pages/admin/admin-management.tsx (hunk 4 above)
Comment (aaggarwal10) [reply pages/admin/admin-management.tsx]: https://github.com/uwblueprint/richmond-centre-for-disability/pull/63/commits/39605957605083d6793db5953bac64ed105a2693

**Review by OustanDing — APPROVED** [review]
LGTM!

**Pull request merged**